packages/types/test/unit/ExecutionAddress.test.ts

```diff
@@ -0,0 +1,61 @@
+import {describe, it, expect} from "vitest";
+import {toChecksumAddress} from "../../src/utils/ExecutionAddress.js";
+
+describe("toChecksumAddress", () => {
+  it("should fail with invalid addresses", () => {
+    expect(() => toChecksumAddress("0x1234")).toThrowError("Invalid address");
```

Comment:
another test case of address without 0x prefix would be good to have (to confirm regex works)